source/statements.tex

```diff
@@ -829,6 +833,50 @@
 before the destruction of local variables\iref{stmt.jump} of the block
 enclosing the \tcode{return} statement.
 
+\rSec3[stmt.return.coroutine]{The \tcode{co_return} statement}%
+\indextext{\idxcode{co_return}}%
+\indextext{coroutine return|see{\tcode{co_return}}}%
+
+\begin{bnf}
+\nontermdef{coroutine-return-statement}\br
+    \terminal{co_return} \opt{expr-or-braced-init-list} \terminal{;}
+\end{bnf}
+
+\pnum
+A coroutine returns to its caller or resumer\iref{dcl.fct.def.coroutine}
+by the \tcode{co_return} statement or when suspended\iref{expr.await}.
+A coroutine shall not return to its caller or resumer
+by a \tcode{return} statement\iref{stmt.return}.
+
+\pnum
+The \grammarterm{expr-braced-init-list} of a \tcode{co_return} statement is called its operand.
```

Comment:
Should be expr-or-braced-init-list I think.

Reply:
Good catch! Fixed by commit b0d6ee5